doc/doxygen/tutorial/tutorial.h.in

```diff
@@ -299,7 +299,7 @@
  *   <tr valign="top">
  *       <td>step-23</td>
  *       <td> Finally a "real" time dependent problem, the wave equation.
- *       Fractional time stepping (explicit, fully implicit and Crank-Nicholson
+ *       Fractional time stepping (explicit, fully implicit and Crank-Nicolson
```

Comment:
Hmm... it seems you can find spelled both ways.

Reply:
Nice! While you're there would you like to also link to that Wikipedia article? The right format would be `[Nicolson](https://en.wikipedia.org/wiki/Phyllis_Nicolson)`. (There is also an article about Crank: https://en.wikipedia.org/wiki/John_Crank .)